stream_test.go

```diff
@@ -52,6 +53,18 @@ var _ = Describe("Stream", func() {
 		resetCalledAtOffset = offset
 	}
 
+	// streamWithTimeout creates an io.ReadWriter that makes sure that Read and Write time out
+	streamWithTimeout := func(str io.ReadWriter) io.ReadWriter {
```

Comment:
The stream is stored in `str` anyway, so this could be another local variable. Then we don't need the function call everywhere.